cmake/external/dml.cmake

```diff
@@ -72,7 +72,7 @@ else()
   if (dml_EXTERNAL_PROJECT)
     set(dml_preset_config $<IF:$<CONFIG:Debug>,debug,release>)
     set(dml_preset_name ${onnxruntime_target_platform}-win-redist-${dml_preset_config})
-    
```

Comment:
So VS Code appears to automatically delete trailing whitespaces now when saving, which is cool, but it makes the diff larger :/. Maybe I can find an option to disable that.